HackingWithSwift/week1/day_5/Writing funktions/Writing functions.playground/Contents.swift

```diff
@@ -12,10 +12,10 @@ func dayForGym () {
 }
 dayForGym()
 
-func PaymantError () {
+func paymantError () {
```

Comment:
1. Remove the extra space after the function name: The function should not have a space between the name and the parentheses.
2. Fix the typo in the function name: The function name should be `paymentError` instead of `paymantError`, following the correct spelling and Swift naming conventions.